test/functional/test_framework/messages.py

```diff
@@ -44,7 +44,7 @@
 
 NODE_NETWORK = (1 << 0)
 # NODE_GETUTXO = (1 << 1)
-NODE_BLOOM = (1 << 2)
```

Comment:
Do not disable this, I will make a test later for allowing NODE_BLOOM services to whitelisted peers.

Reply:
No need to invalidate reviews. Just uncomment it in your PR.

Reply:
(also linting tools like vulture may flag an error if this constant is defined but not used)

Reply:
That sounds like a bug in the linting tool IMO.